src/notebooks/telemetry/interpreterPackageTracker.ts

```diff
@@ -3,16 +3,16 @@
 
 import { inject, injectable, optional } from 'inversify';
 import { NotebookDocument } from 'vscode';
-import { IExtensionSingleActivationService } from '../platform/activation/types';
-import { IPythonExtensionChecker, IPythonApiProvider } from '../platform/api/types';
-import { IExtensions, IDisposableRegistry, InterpreterUri } from '../platform/common/types';
-import { isResource, noop } from '../platform/common/utils/misc';
-import { IInterpreterService } from '../platform/interpreter/contracts';
-import { INotebookControllerManager } from '../notebooks/types';
-import { IInstaller, Product } from '../kernels/installer/types';
-import { IVSCodeNotebookController } from '../notebooks/controllers/types';
-import { trackKernelResourceInformation } from './telemetry';
-import { IInterpreterPackages } from './types';
+import { IExtensionSingleActivationService } from '../../platform/activation/types';
+import { IPythonExtensionChecker, IPythonApiProvider } from '../../platform/api/types';
+import { IExtensions, IDisposableRegistry, InterpreterUri } from '../../platform/common/types';
+import { isResource, noop } from '../../platform/common/utils/misc';
+import { IInterpreterService } from '../../platform/interpreter/contracts';
+import { INotebookControllerManager } from '../types';
+import { IInstaller, Product } from '../../kernels/installer/types';
+import { IVSCodeNotebookController } from '../controllers/types';
+import { trackKernelResourceInformation } from '../../kernels/telemetry/helper';
```

Comment:
A side effect is, everything that didn't depend on kernels, now pulls in dependency from kernels folder.

Reply:
I guess that's what we're trying to fix here, and this PR is just one of many.

Reply:
@DonJayamanne you are right. This particular module `InterpreterPackageTracker` needs to read `INotebookControllerManager`, `./src/kernels`, and `./src/telemetry`. The best place for it is probably out of`./src/notebooks` and placed in a folder where we have all the moduels that glue things together.